docs/en/tutorials/runner.md

```diff
@@ -1,3 +1,523 @@
 # Runner
 
-Coming soon. Please refer to [chinese documentation](https://mmengine.readthedocs.io/zh_CN/latest/tutorials/runner.html).
+Welcome to the tutorial of runner, the core of MMEngine's user interface!
+
+The runner, as an "integrator" in MMEngine, covers all aspects of the framework and shoulders the responsibility of organizing and scheduling nearly all modules. Therefore, the code and implementation logic in it have to take into account various situations, making it relatively hard to understand. But **don't worry**! In this tutorial, we will leave out some messy details and have a quick overview of commonly used APIs, functionalities and examples. Hopefully this should provide you with a clear and easy-to-understand user interface. After reading through this tutorial, you will be able to:
+
+- Master the common usage and configuration of the runner
+- Learn the best practice - writing config files - of the runner
+- Know about the basic dataflow and execution order
+- Feel by yourself the advantages of using runner (perhaps)
+
+## Example codes of the runner
+
+To build your training pipeline with a runner, there are typically two ways to get started:
+
+- Refer to runner's [API documentations](mmengine.runner.Runner) for argument-by-argument configuration
+- Make your custom modifications based on some existing configurations, such as [Getting started in 15 minutes](../get_started/15_minutes.md) and downstream repos like [MMDet](https://github.com/open-mmlab/mmdetection)
+
+Pros and cons lie in both approaches. For the former one, beginners may be lost in a vast number of configurable arguments. For the latter one, beginers may find it hard to get a good reference, since neither an over-simplified nor an over-detailed reference is conductive to them.
+
+We argue that the key to learn runner is using it as a memo. You should remember its most commonly used arguments and only focus on those less used when in need, since default values usually work fine. In the following we will provide a beginer-friendly example to illustrate most commonly used arguments of the runner, along with advanced guidelines for those less used.
```

Comment:
```suggestion
To build your training pipeline with a runner, there are typically two ways to get started:

- Refer to runner's [API documentation](mmengine.runner.Runner) for argument-by-argument configuration
- Make your custom modifications based on some existing configurations, such as [Getting started in 15 minutes](../get_started/15_minutes.md) and downstream repositories like [MMDet](https://github.com/open-mmlab/mmdetection)

Pros and cons lie in both approaches. For the former one, beginners may be lost in a vast number of configurable arguments. For the latter one, beginners may find it hard to get a good reference, since neither an over-simplified nor an over-detailed reference is conducive to them.

We argue that the key to learning runner is using it as a memo. You should remember its most commonly used arguments and only focus on those less used when in need, since default values usually work fine. In the following, we will provide a beginner-friendly example to illustrate the most commonly used arguments of the runner, along with advanced guidelines for those less used.
```